cmd/image-inspector.go

```diff
@@ -28,6 +29,8 @@ func main() {
 
 	flag.Parse()
 
+	inspectorOptions.AuthToken = os.Getenv("INSPECTOR_AUTH_TOKEN")
```

Comment:
@mfojtik @ilackarms any way to define a secret in the Pod definition? I hate to think that ManageIQ should also orchestrate this secret as a separate object.